docs/src/modules/ROOT/pages/optimization-algorithms/overview.adoc

```diff
@@ -565,6 +565,58 @@ Terminates as soon as a feasible solution has been discovered.
 
 This `Termination` is usually combined with other terminations.
 
+[#unimprovedBestSolutionTermination]
+==== `UnimprovedBestSolutionTermination`
+
+`UnimprovedBestSolutionTermination` terminates when the solver cannot improve the best solution in an estimated time interval.
+The estimation of the termination interval is based on the last time
+the solver improved the solution and how much time has passed without any improvements.
+
+[source,xml,options="nowrap"]
+----
+  <termination>
+    <unimprovedBestSolutionLimit>0.5</unimprovedBestSolutionLimit>
+  </termination>
+----
+
+Let's consider a solution process based on the following image.
+
+image::optimization-algorithms/overview/terminationGrowthCurve.png[align="center"]
+
+In the previous image, the estimated unimproved time is represented by the flat red line.
+With a configuration for `unimprovedBestSolutionLimit` set to `0.5`,
+and considering that the last best score was achieved at `10 seconds`,
+the solver will terminate the process if no improvement is found within a maximum of 10 seconds.
+This `10-seconds` limit is calculated
+by multiplying the estimated total time (`20 seconds`) by the `unimprovedBestSolutionLimit` (`0.5`).
```

Comment:
Isn't this a bit confusing?
Why not change the definition and instead of basing it on estimated total time, it would be based on the time it spent improving?

The limit would be `1.0` in that case, and the docs would say eg. "the last best score was achieved at 10 seconds, and the solver will terminate in another 10 seconds."